sympy/tensor/tensor.py

```diff
@@ -44,6 +45,13 @@
 from sympy.core.sympify import CantSympify, _sympify
 from sympy.core.operations import AssocOp
 from sympy.matrices import eye
+from sympy.utilities.exceptions import SymPyDeprecationWarning
+import warnings
+
+
+@deprecated(useinstead=".replace_with_arrays", issue=15276, deprecated_since_version="1.4")
```

Comment:
Please use new "DeprecationRemoval" issues for the issue in deprecation warnings. See https://github.com/sympy/sympy/wiki/Deprecating-policy.